### PR TITLE
docs: mention that backup is skipped

### PR DESCRIPTION
**Pull request by hasbro17**
[skip ci]

For issues like #941 

/cc @xiang90 @hongchaodeng

### CODE REVIEW

File: doc/user/backup_service.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ## Backup service
 
 A backup service will be created if the etcd cluster has backup enabled.
-The backup service saves backup for the etcd cluster based on the requirement of the backup spec.
+The backup service saves backup for the etcd cluster based on the requirement of the [backup spec](https://github.com/coreos/etcd-operator/blob/master/example/example-etcd-cluster-with-backup.yaml#L8-L12).
+
+The backup service will skip creating a new snapshot if the etcd `Revision` has not changed since the last snapshot, i.e the etcd-cluster data has not been modified (e.g., `Put`, `Delete`, `Txn`).
```
Comment (xiang90) [inline doc/user/backup_service.md]: skip creating new snapshot if (remove `a`?)

/cc @heyitsanthony

**Comment by xiang90** [comment]
lgtm

File: doc/user/backup_service.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ## Backup service
 
 A backup service will be created if the etcd cluster has backup enabled.
-The backup service saves backup for the etcd cluster based on the requirement of the backup spec.
+The backup service saves backup for the etcd cluster based on the requirement of the [backup spec](https://github.com/coreos/etcd-operator/blob/master/example/example-etcd-cluster-with-backup.yaml#L8-L12).
```
Comment (hongchaodeng) [inline doc/user/backup_service.md]: Don't use `blob/master`.
Use git hash.

File: doc/user/backup_service.md (hunk 1 above)
Comment (hongchaodeng) [inline doc/user/backup_service.md]: if the etcd \`Revision\` -> if the etcd cluster revision

Comment (hasbro17) [inline doc/user/backup_service.md]: I think the `a` should be there grammar wise.

**Pull request merged**